python/ml4ir/applications/classification/pipeline.py

```diff
@@ -160,6 +160,16 @@ def get_relevance_dataset(
 
         return relevance_dataset
 
+    def create_pipeline_for_kfold(self, args):
+        """
+        Create a ClassificationPipeline object used in running kfold cross validation.
+        """
+        return ClassificationPipeline(args=args)
+
+    def run_kfold_analysis(self, base_logs_dir, base_run_id, num_folds):
+        # TODO: Implement the kfold CV analysis for classification
```

Comment:
@mohazahran What is stopping us from doing this? I would like this to be generic enough for all ml4ir pipelines ideally.

Reply:
As discussed in the call, for ranking pipeline, I compare old vs. new models with ttest. Classification -as far as I know- does not have old, new model results.